packages/docs-theme/src/components/typescript/apiHeader.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the terms of the LICENSE file distributed with this project.
+ */
+
+import { isTsClass, isTsInterface, ITsDocBase } from "documentalist/dist/client";
+import * as React from "react";
+import { DocumentationContextTypes, IDocumentationContext } from "../../common/context";
+
+export const ApiHeader: React.SFC<ITsDocBase> = (data, { renderType }: IDocumentationContext) => {
+    const inheritance =
+        isTsClass(data) || isTsInterface(data)
+            ? maybeJoinArray("extends", data.extends) + " " + maybeJoinArray("implements", data.implements)
+            : "";
+    return (
+        <div className="docs-interface-header">
+            <div className="docs-interface-name">
+                <small>{data.kind}</small> {data.name} <small>{renderType(inheritance)}</small>
+            </div>
+            <small className="docs-package-name">
+                <a href={GITHUB_URL + data.fileName.slice(3)} target="__blank">
+                    @blueprintjs/{data.fileName.split("/", 2)[1]}
```

Comment:
this needs to be configurable... you're in the docs-theme package, not docs-app

Reply:
oh snap so true. i think i need to add a `url` field to the DM data... typedoc already provides one that even links to line number.

Reply:
oops this is not done. not sure how to support it either...